db/migrate/20210801104200_create_storage_hosts_wwpn_candidates.rb

```diff
@@ -0,0 +1,14 @@
+class CreateStorageHostsWwpnCandidates < ActiveRecord::Migration[6.0]
+  def change
+    create_table :storage_hosts_wwpn_candidates do |t|
```

Comment:
Since this belongs to a physical_storage might `physical_storage_wwpn_candidates` be a better name?

Reply:
will change to this name

Reply:
@ereztheodorou the `storage_hosts` still is confusing though because we don't have that name anywhere in our database and the current table name is in the format of a rails join table (which would normally indicate that it joined storage_hosts model).

Reply:
@ereztheodorou @agrare so maybe simply the 'wwpn_candidates' will go?

Reply:
Yeah that works if you don't want the physical_storage prefix